### DISCUSSION TITLE
Battery reporting changed?

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js#4626] [General]
Hey guys, I'm on ZWave-JS v 9.0.7 (latest ver of zw2m v6.8.1), and I've noticed recently more of my battery powered devices (Zooz ZSE40 Multisensors) reporting 0% battery levels, even though they've been changed recently and the device is clearly reporting Temp, Humidity, Luminance and Motion correctly. In the past I've had devices be completely dead even though the battery showed sometimes 50-90% levels when they died, but I've never had a LIVE unit reporting a battery at 0%. I now have TWO.

Has anything changed recently that may have modified this reporting behavior, or interpretation of the data reported? It seems out of the ordinary.

**Comment by AlCalzone** [comment]
Nothing changed. Might be worth looking at the driver logs to see what the devices report. I've had a sensor with vastly varying battery reports before, maybe this is a measurement fluke on the node.

**Reply by GrizzlyAK** [reply]
I'll take a look at that. I don't rely too heavily on battery reports because none of them seem reliable to me. I use template sensors that look at drop offs in sensor reporting (eg, temp), but that doesn't work for things like tilt sensors or door sensors that aren't used regularly. It looks like Frank has just updated the HA ZW2M Add-on which  includes the latest zwave stuff. Not released just yet. Just thought the frequency all of a sudden may be an artifact of some change. Only affecting the ZSE40s atm. Thanks.
